packages/apps/src/configs/wallets/chain-config.ts

```diff
@@ -1,13 +1,15 @@
 import { AppConfig } from '@webb-dapp/react-environment/webb-context';
 import EdgwareLogo from '@webb-dapp/apps/configs/wallets/logos/EdgwareLogo';
 import EtherLogo from '@webb-dapp/apps/configs/wallets/logos/Eth';
+import { WebbEVMChain } from '@webb-dapp/react-environment/api-providers/web3';
 
 export const chainsConfig: AppConfig['chains'] = {
   // edgware
   1: {
     id: 1,
     group: 'edgware',
     tag: 'dev',
+    evmId: undefined,
```

Comment:
If we use enums then we should make sure:
- Edgeware mainnet is 2021
- Edgeware beresheet is 2022

Reply:
Will handle this on a refactor PR after the #111